src/compiler/chainrules.jl

```diff
@@ -128,6 +128,39 @@ Convert `x` from the format Zygote uses internally to differentials types ChainR
   ChainRules.Tangent{Any, typeof(xp)}(xp)
 end
 
+"""
+  _project(x)(dx)
```

Comment:
This sound like it should be in CRC, not zygote.

cc @oxinabox

Reply:
If you check the CRC issues, you will see much discussion of this exact point. WIP, as it says.